patches/buildroot/0006-Add-patch-series-for-stack-protection.patch

```diff
@@ -2686,7 +2686,7 @@ index 0000000000..d5ab90340d
 +
 diff --git a/package/gcc/11.3.0/9019-aarch64-Make-stack-smash-canary-protect-saved-registers.patch b/package/gcc/11.3.0/9019-aarch64-Make-stack-smash-canary-protect-saved-registers.patch
 new file mode 100644
-index 0000000000..33866e8afd
+index 00000000..33866e8a
 --- /dev/null
 +++ b/package/gcc/11.3.0/9019-aarch64-Make-stack-smash-canary-protect-saved-registers.patch
```

Comment:
This patch series ends up being lost because GCC was updated from 11.3.0 to 11.4.0, so the patch files aren't picked up during the build.